jdbc-driver/build.gradle

```diff
@@ -0,0 +1,44 @@
+plugins {
```

Comment:
Move `src/` folder in the project root and delete `jdbc-driver/` folder and all duplicate files that are already in the project root?

Reply:
Our intention was to create a multi module build including a shaded Hive artifact, the main driver, and an example client. So I think this is fine as is

Reply:
Current setup doesn't look like multi-project Gradle build: https://guides.gradle.org/creating-multi-project-builds/

So, I guess we need to fix it?

Reply:
Yeah, sgtm

Reply:
Reorganized the project to be multi-project Gradle.